src/test/scala/vsys/blockchain/contract/channel/PaymentChannelContractGen.scala

```diff
@@ -0,0 +1,98 @@
+package vsys.blockchain.contract.channel
+
+import com.google.common.primitives.Longs
+import org.scalacheck.Gen
+import vsys.account.{Address, ContractAccount, PrivateKeyAccount}
+import vsys.blockchain.contract.ContractGenHelper._
+import vsys.blockchain.contract._
+import vsys.blockchain.state._
+import vsys.blockchain.transaction.GenesisTransaction
+import vsys.blockchain.transaction.contract.{ExecuteContractFunctionTransaction, RegisterContractTransaction}
+
+trait PaymentChannelContractGen {
+
+  val createIndex: Short = 0
+  val updateExpiredTimeIndex: Short = 1
+  val chargeIndex: Short = 2
+  val terminateIndex: Short = 3
+  val executeWithdrawIndex: Short = 4
+  val executePaymentIndex: Short = 5
+
+  def paymentChannelContractGen(): Gen[Contract] =
+    ContractPaymentChannel.contract
+
+  def initPaymentChannelContractDataStackGen(tokenId: Array[Byte]): Gen[Seq[DataEntry]] = for {
+    tokenId <- Gen.const(DataEntry(tokenId, DataType.TokenId))
+  } yield Seq(tokenId)
+
+  def depositChannelContractDataStackGen(depositor: Address, amount: Long, tokenId: Array[Byte]): Gen[Seq[DataEntry]] = for {
+    deposit <- Gen.const(DataEntry(depositor.bytes.arr, DataType.Address))
+    res <- Gen.const(DataEntry(Longs.toByteArray(amount), DataType.Amount))
+    tokenId <- Gen.const(DataEntry(tokenId, DataType.TokenId))
+  } yield Seq(deposit, res, tokenId)
+
+  def withdrawChannelContractDataStackGen(withdrawer: Address, amount: Long, tokenId: Array[Byte]): Gen[Seq[DataEntry]] = for {
+    withdraw <- Gen.const(DataEntry(withdrawer.bytes.arr, DataType.Address))
+    res <- Gen.const(DataEntry(Longs.toByteArray(amount), DataType.Amount))
+    tokenId <- Gen.const(DataEntry(tokenId, DataType.TokenId))
+  } yield Seq(withdraw, res, tokenId)
```

Comment:
Since deposit/withdraw triggers are called by token/system contract, so it seems unused here.